app/controllers/recipe_controller.rb

```diff
@@ -1,2 +1,37 @@
 class RecipeController < ApplicationController
+  def new
+    @recipe = Recipe.new
+    @ingredients = Ingredient.all
```

Comment:
Why are we setting the instance variable here for `all` Ingredients?

Would something like `@ingredients = @recipe.ingredients` make more sense so the ingredients we're instantiating for the view are scoped to the specific recipe we're viewing? 🤔

Reply:
^ yeah that makes way more sense